components/02-molecules/text-with-media/_text-with-media.scss

```diff
@@ -0,0 +1,16 @@
+.text-with-media__inner {
```

Comment:
@josue2591 , hey I didn't notice this! I'm actually not sure if this is expected behavior! I will ask to see if it is. Intuitively I feel like we should have a "fixed" media width, but since I'm not sure I'll ask my manager. Thank you for catching this one, and I'll update this thread with an answer when I have one :)!

Reply:
@callinmullaney , Hey Callin is this expected behavior? I'm thinking no?

Reply:
I would say this behavior is expected but not ideal. However, because we are trying to implement as little styling assumptions as possible I think this passes.